tools/meteor-services/deploy.js

```diff
@@ -442,8 +644,20 @@ export function bundleAndDeploy(options) {
     return 1;
   }
 
+  // After an upload succeeds, we want to poll Galaxy to see if the build / deploy succeeds
+  // We used to indicate that the upload was finished via a message to check Galaxy
   if (result.payload.message) {
-    Console.info(result.payload.message);
+    // If there is a newVersionId, then poll for status of the app
```

Comment:
I'm not sure why this is inside the `if message` conditional if it doesn't read the message in the new path.

Reply:
In fact, I'll have it always print the message if it's there. That way, we can maintain the idea you were talking about of having a way to report messages to Meteor users without changing Meteor.